python/ray/tests/test_basic.py

```diff
@@ -163,6 +176,7 @@ class A2:
                 x = 1
 
 
+@pytest.mark.skipif(client_test_enabled(), reason="internal _remote")
```

Comment:
This is actually a pretty important test, since it covers a lot of the custom resources handling subtlety in the Ray API. We should make sure to enable it once the resource kwargs are supported.

Btw you can change `_remote(args, remote_args)` to use `foo.options(remote_args).remote(args)` (it's the new syntax). Not sure if client supports `.options()` right now.

Reply:
I see. So this would follow under the "remote args" category -- which is a target for the things that I'm working on right this second in supporting `.options`

So I'll change the reason.